Src/Plugins/SalModernization/SalModernization.SharedStrings.txt

```diff
@@ -21,363 +21,363 @@
-$SEC105/002.Inoutzcapc=[^\w_]_Inout_z_cap_c_[^\w_]
-$SEC105/002.Outbytecap=[^\w_]_Out_bytecap_[^\w_]
-$SEC105/002.Outbytecapc=[^\w_]_Out_bytecap_c_[^\w_]
-$SEC105/002.Outbytecappostbytecount=[^\w_]_Out_bytecap_post_bytecount_[^\w_]
-$SEC105/003.Outbytecapx=[^\w_]_Out_bytecap_x_[^\w_]
-$SEC105/002.Outbytecapcount=[^\w_]_Out_bytecapcount_[^\w_]
-$SEC105/002.Outcap=[^\w_]_Out_cap_[^\w_]
-$SEC105/002.Outcapc=[^\w_]_Out_cap_c_[^\w_]
-$SEC105/003.Outcapm=[^\w_]_Out_cap_m_[^\w_]
-$SEC105/002.Outcappostcount=[^\w_]_Out_cap_post_count_[^\w_]
-$SEC105/003.Outcapx=[^\w_]_Out_cap_x_[^\w_]
-$SEC105/002.Outcapcount=[^\w_]_Out_capcount_[^\w_]
-$SEC105/003.Outcapcountx=[^\w_]_Out_capcount_x_[^\w_]
-$SEC105/002.Outoptbytecap=[^\w_]_Out_opt_bytecap_[^\w_]
-$SEC105/002.Outoptbytecappostbytecount=[^\w_]_Out_opt_bytecap_post_bytecount_[^\w_]
-$SEC105/002.Outoptbytecapcount=[^\w_]_Out_opt_bytecapcount_[^\w_]
-$SEC105/002.Outoptcap=[^\w_]_Out_opt_cap_[^\w_]
-$SEC105/002.Outoptcapc=[^\w_]_Out_opt_cap_c_[^\w_]
-$SEC105/003.Outoptcapx=[^\w_]_Out_opt_cap_x_[^\w_]
-$SEC105/002.Outoptcappostcount=[^\w_]_Out_opt_cap_post_count_[^\w_]
-$SEC105/002.Outoptcapcount=[^\w_]_Out_opt_capcount_[^\w_]
-$SEC105/003.Outoptzbytecap=[^\w_]_Out_opt_z_bytecap_[^\w_]
-$SEC105/003.Outoptzbytecapc=[^\w_]_Out_opt_z_bytecap_c_[^\w_]
-$SEC105/002.Outoptzcap=[^\w_]_Out_opt_z_cap_[^\w_]
-$SEC105/003.Outoptzcappostcount=[^\w_]_Out_opt_z_cap_post_count_[^\w_]
-$SEC105/003.Outzbytecap=[^\w_]_Out_z_bytecap_[^\w_]
-$SEC105/003.Outzbytecapc=[^\w_]_Out_z_bytecap_c_[^\w_]
-$SEC105/002.Outzcap=[^\w_]_Out_z_cap_[^\w_]
-$SEC105/002.Outzcapc=[^\w_]_Out_z_cap_c_[^\w_]
-$SEC105/003.Outzcappostcount=[^\w_]_Out_z_cap_post_count_[^\w_]
-$SEC105/003.Outzcapx=[^\w_]_Out_z_cap_x_[^\w_]
-$SEC105/003.Outzcapcount=[^\w_]_Out_z_capcount_[^\w_]
-$SEC105/002.Postbytecount=[^\w_]_Post_bytecount_[^\w_]
-$SEC105/002.Postbytecountc=[^\w_]_Post_bytecount_c_[^\w_]
-$SEC105/003.Postbytecountx=[^\w_]_Post_bytecount_x_[^\w_]
-$SEC105/002.Postcount=[^\w_]_Post_count_[^\w_]
-$SEC105/002.Postcountc=[^\w_]_Post_count_c_[^\w_]
-$SEC105/002.Postzcount=[^\w_]_Post_z_count_[^\w_]
-$SEC105/003.Precapfor=[^\w_]_Pre_cap_for_[^\w_]
-$SEC105/002.Precount=[^\w_]_Pre_count_[^\w_]
-$SEC105/003.Preoptbytecountx=[^\w_]_Pre_opt_bytecount_x_[^\w_]
-$SEC105/003.Prepostbytecountx=[^\w_]_Prepost_bytecount_x_[^\w_]
-$SEC105/003.Prepostcount=[^\w_]_Prepost_count_[^\w_]
-$SEC105/003.Prepostoptbytecountx=[^\w_]_Prepost_opt_bytecount_x_[^\w_]
-$SEC105/002.Retbytecap=[^\w_]_Ret_bytecap_[^\w_]
-$SEC105/002.Retcap=[^\w_]_Ret_cap_[^\w_]
-$SEC105/003.Retcountx=[^\w_]_Ret_count_x_[^\w_]
-$SEC105/002.Retoptbytecap=[^\w_]_Ret_opt_bytecap_[^\w_]
-$SEC105/003.Retoptbytecapx=[^\w_]_Ret_opt_bytecap_x_[^\w_]
-$SEC105/003.Retoptbytecountx=[^\w_]_Ret_opt_bytecount_x_[^\w_]
-$SEC105/002.Retoptcount=[^\w_]_Ret_opt_count_[^\w_]
-$SEC105/002.Retoptzcap=[^\w_]_Ret_opt_z_cap_[^\w_]
+$SEC105/001.allocator=[^\w_](?P<refine>__allocator)[^\w_]
```

Comment:
>(?P<refine> [](http://example.com/codeflow?start=28&length=11)

This is the only change to all of these, correct?

Reply:
Correct.
The reason why I didn't add this in all lines is because not all have separators.

Reply:
good to know the predefined/preserved group name `refine` can be used in this case 😄